building/devices/nuvoton.rst

```diff
@@ -0,0 +1,92 @@
+.. _nuvoton:
+
+#######
+NUVOTON
+#######
+
+The instructions here will tell how to build and run OP-TEE OS
+for Nuvoton platform standalone (not as a part of openbmc image).
+
+
+.. _nuvoton_build_instructions:
+
+Build instructions
+******************
+
+=================
+Pre-requirements:
+=================
+
+#. Download and install the latest IGPS from the Nuvoton Israel GitHub
+
+    .. code-block:: bash
+
+        $ git clone https://github.com/Nuvoton-Israel/igps-npcm8xx
+
+#. Download and extract the latest Linux based Arm GNU Toolchain
+for aarch64 bare-metal target, for example, 12.2
+
+    .. code-block:: bash
+
+        $ cd /opt
+        $ wget https://developer.arm.com/-/media/Files/downloads/gnu/12.2.rel1/binrel/arm-gnu-toolchain-12.2.rel1-x86_64-aarch64-none-elf.tar.xz?rev=28d5199f6db34e5980aae1062e5a6703&hash=D87D4B558F0A2247B255BA15C32A94A9F354E6A8
+        $ tar xvf arm-gnu-toolchain-12.2.rel1-x86_64-aarch64-none-elf.tar.xz
+
+#. Add the Arm GNU Toolchain binary to the $PATH
+
+    .. code-block:: bash
+
+        $ cd ~
+        $ vi .bashrc
+        export PATH=$PATH:/opt/arm-gnu-toolchain-12.2.rel1-x86_64-aarch64-none-elf/bin
+
+#. Clone the latest OP-TEE OS code from the github
+
+    .. code-block:: bash
+
+        $ git clone https://github.com/OP-TEE/optee_os.git
+        $ cd optee_os
+
+==============
+Build process:
+==============
+
+#. build OP-TEE OS for Nuvoton platform
+
+    .. code-block:: bash
+
+        $ make CROSS_COMPILE64=aarch64-none-elf- PLATFORM=nuvoton -j
+
+   Note: you can use additional debug flag for compilation to get debug prints to console
+
+    .. code-block:: bash
+
+        $ make CROSS_COMPILE64=aarch64-none-elf- PLATFORM=nuvoton CFG_NPCM_DEBUG=y -j $(nproc)
+
+#. Update binary input files in IGPS
+
+    .. code-block:: bash
+
+        python ./UpdateInputsBinaries_Arbel_A1_EB.py
```

Comment:
I think the reference to local current directory `./` here is wrong at this stage, at least confusing.
Maybe instruction `cd /path/to/igps-npcm8xx` is missing.